driver-core/build.gradle

```diff
@@ -31,17 +31,17 @@ ext {
 sourceSets.main.resources.srcDirs = ['src/resources']
 
 dependencies {
-    compile project(path: ':bson', configuration: 'default')
+    api project(path: ':bson', configuration: 'default')
 
-    compile "com.github.jnr:jnr-unixsocket:$jnrUnixsocketVersion", optional
-    compile "io.netty:netty-buffer:$nettyVersion", optional
-    compile "io.netty:netty-transport:$nettyVersion", optional
-    compile "io.netty:netty-handler:$nettyVersion", optional
-    compile "org.xerial.snappy:snappy-java:$snappyVersion", optional
-    compile "com.github.luben:zstd-jni:$zstdVersion", optional
-    compile "org.mongodb:mongodb-crypt:$mongoCryptVersion", optional
+    implementation "com.github.jnr:jnr-unixsocket:$jnrUnixsocketVersion", optional
+    implementation "io.netty:netty-buffer:$nettyVersion", optional
+    implementation "io.netty:netty-transport:$nettyVersion", optional
+    implementation "io.netty:netty-handler:$nettyVersion", optional
```

Comment:
I think this is all implementation, but not positive.

Reply:
Switching to `api`.

Reply:
What types from netty-handler are exposted in our public API?

Reply:
Agreed on using `implementation`.